app/lib/omni_auth/strategies/dev_auth.rb

```diff
@@ -0,0 +1,67 @@
+module OmniAuth
+  module Strategies
+    class DevAuth
+      # IMPORTANT NOTE: This OmniAuth strategy is intended for local
+      # development purposes only.
+      #
+      # When a user clicks the "Sign in" button with this strategy,
```

Comment:
Wondering ig this should go into the README rather than here?

Reply:
done